### ISSUE TITLE
do we still need to generate kubeconfig files

### ISSUE THREAD

**Issue by dustymabe** [projectatomic/atomicapp#347]
We started generating a kube config file for https://github.com/projectatomic/atomicapp/issues/280 because it looks like kubernetes would complain if ~/.kube/config didn't exist. It no longer complains about this (in latest kubernetes in Fedora and CentOS). Should we drop this error prone behavior? 

Here is some audit output from me running kubectl get pods on my machine:
```
#    date        time      file                                                 syscall  success  exe               auid     event
1.   10/20/2015  15:58:13  /bin/kubectl                                         execve   yes      /usr/bin/kubectl  vagrant  724
2.   10/20/2015  15:58:13  /etc/ld.so.preload                                   access   no       /usr/bin/kubectl  vagrant  727
3.   10/20/2015  15:58:13  /etc/ld.so.cache                                     open     yes      /usr/bin/kubectl  vagrant  728
4.   10/20/2015  15:58:13  /lib64/libpthread.so.0                               open     yes      /usr/bin/kubectl  vagrant  732
5.   10/20/2015  15:58:13  /lib64/libc.so.6                                     open     yes      /usr/bin/kubectl  vagrant  740
6.   10/20/2015  15:58:13  /proc/sys/kernel/hostname                            open     yes      /usr/bin/kubectl  vagrant  844
7.   10/20/2015  15:58:13  /proc/sys/net/core/somaxconn                         open     yes      /usr/bin/kubectl  vagrant  850
8.   10/20/2015  15:58:13  /proc                                                stat     yes      /usr/bin/kubectl  vagrant  872
9.   10/20/2015  15:58:13  /proc/stat                                           open     yes      /usr/bin/kubectl  vagrant  873
10.  10/20/2015  15:58:13  /root/.kube/config                                   stat     no       /usr/bin/kubectl  vagrant  895
11.  10/20/2015  15:58:13  /root/.kube/.kubeconfig                              stat     no       /usr/bin/kubectl  vagrant  896
12.  10/20/2015  15:58:13  /root/.kube/config                                   open     no       /usr/bin/kubectl  vagrant  897
13.  10/20/2015  15:58:13  /root/.kube/config                                   open     no       /usr/bin/kubectl  vagrant  898
14.  10/20/2015  15:58:13  /var/run/secrets/kubernetes.io/serviceaccount/token  stat     no       /usr/bin/kubectl  vagrant  899
15.  10/20/2015  15:58:13  /root/.kube/config                                   stat     no       /usr/bin/kubectl  vagrant  900
16.  10/20/2015  15:58:13  /root/.kube/.kubeconfig                              stat     no       /usr/bin/kubectl  vagrant  901
17.  10/20/2015  15:58:13  /root/.kube/config                                   open     no       /usr/bin/kubectl  vagrant  902
18.  10/20/2015  15:58:13  /root/.kube/config                                   open     no       /usr/bin/kubectl  vagrant  903
19.  10/20/2015  15:58:13  /root/.kube/config                                   stat     no       /usr/bin/kubectl  vagrant  904
20.  10/20/2015  15:58:13  /root/.kube/.kubeconfig                              stat     no       /usr/bin/kubectl  vagrant  905
21.  10/20/2015  15:58:13  /root/.kube/config                                   open     no       /usr/bin/kubectl  vagrant  906
22.  10/20/2015  15:58:13  /root/.kube/config                                   open     no       /usr/bin/kubectl  vagrant  907
23.  10/20/2015  15:58:13  /var/run/secrets/kubernetes.io/serviceaccount/token  stat     no       /usr/bin/kubectl  vagrant  908
24.  10/20/2015  15:58:13  /etc/hosts                                           open     yes      /usr/bin/kubectl  vagrant  909
```

As you can see none of the attempts to open a config file succeeded and I was not presented with an error.

**Comment by dustymabe** [comment]
@vpavlin Since you implemented #280 ^^

**Comment by dustymabe** [comment]
Fixed by #348
